### ISSUE TITLE
Fine-tune parameters and sticker datasets

### ISSUE THREAD

**Issue by junyizeng** [jinyugy21/Adv-Stickers_RHDE#4]
Thank you for your reply. Can you provide some specific parameters of your finetune? (For example, the loss of fine-tune, fine-tune step size, learning rate, optimizer parameters) In addition, other stickers are used in your paper. Could you provide the sticker data set you used? Or the way to get stickers.

**Comment by jinyugy21** [comment]
The model can be trained in the normal way, and no special parameter tuning skills are required. When the val acc is above 99 on the LFW, it can be stopped. 
If the sticker is a real sticker with relatively rich graphic content, it should be fine, and if the sticker contains some cartoon information for the five senses, the effect may be better.

**Comment by junyizeng** [comment]
Thank you very much for your reply. When I was reproducing the project, I found that your sticker image is an RGBD image, and I tried to remove the depth layer of the image and use the corresponding RGB image, but the resulting image is particularly unnatural. Do you use the RGBD image directly for the sticker data or generate the depth layer based on the RGB image? If so, can you share with the code of generating the depth layer or the source of the sticker dataset?

**Comment by jinyugy21** [comment]
Do you mean 'RGBA'? https://github.com/jinyugy21/Adv-Stickers_RHDE/blob/53a16c8f90aa7f779a09eb7669941279311f6053/utils/stick.py#L23
We used a four-channel sticker, also known as PNG format, where the alpha channel controls the transparency of the image.
